test/uk/gov/hmrc/r2/selfassessmentapi/models/selfemployment/AdjustmentsSpec.scala

```diff
@@ -109,8 +114,13 @@ class AdjustmentsSpec extends JsonSpec {
       validateAmount(se, "/averagingAdjustment")
     }
 
-    "reject averagingAdjustment more than 99999999999999.98" in {
-      val se = Adjustments(averagingAdjustment = Some(BigDecimal("99999999999999.99")))
+    "reject averagingAdjustment more than 99999999999.99" in {
+      val se = Adjustments(averagingAdjustment = Some(BigDecimal("999999999999.99")))
+      validateAmount(se, "/averagingAdjustment")
+    }
+
+    "reject averagingAdjustment less than -99999999999.99" in {
+      val se = Adjustments(averagingAdjustment = Some(BigDecimal("-999999999999.99")))
```

Comment:
Likewise those testing the lower limit:
```suggestion
      val se = Adjustments(averagingAdjustment = Some(BigDecimal("-100000000000")))
```